tests_in_tilt/Tiltfile

```diff
@@ -0,0 +1,51 @@
+# Some common test configurations to easily get up and running with Tilt for Tests
+# (see docs: https://docs.tilt.dev/tests_in_tilt.html)
+
+### GOLANG TESTS ###
+def test_go_all(path, deps, timeout='', tags=None, mod=''):
```

Comment:
Instead of passing `deps` (or if user passes empty `deps`) we could find all `*.go` / `*.js||tsx` files, but testing this in our repo, it takes ~3 seconds to `find` all go files, so probably not worth it?

Reply:
* Even if it were instantaneous, `find -name "*.go"` is kinda annoying in that if I create a new go file after the Tiltfile executes, changes to it won't trigger a build of this resource.
* You could let the user decide by documenting an example that calls `find` or even add a `find` function and have an example call that to set deps.